projects/gen-ai.md

```diff
@@ -0,0 +1,157 @@
+
+OpenLLMetry provides general-purpose OTel instrumentations for GenAI clients, frameworks and related components; they are governed by [Traceloop](https://www.traceloop.com/docs/introduction).
+
+By providing GenAI community with the space under OpenTelemetry we will be able to start hosting vendor-agnostic instrumentation libraries (such as WIP one [OpenAI Python](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2759)) and eventually we will be able to move popular OpenLLMetry-owned `opentelemetry-instrumentation-*` libraries to OTel (or to upstream libraries).
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following exiting conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with OpenLLMetry for a set of client instrumentation libraries that all vendors can depends upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and LLM frameworks for popular libraries in different language
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel LLM SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
```

Comment:
What's the difference between Project Lead here and the current Maintainer/Approver roles in the LLM Semver SIG? Is the new maintainer role the product of old plus this, or is this lead role something entirely different?

Reply:
there is no formal definition of the project lead role, that's how it's described in the project template 

https://github.com/open-telemetry/community/blob/ad757c73a375e59211ae0986f707f6e9de76d848/project-template.md?plain=1#L26


in practice project leads usually run meetings, update project board, do process-related stuff and anything else necessary (as the note above suggests). Some responsibility, but no special permissions.

Reply:
ok I guess leads here are not directly related to the [project it is expanding](https://github.com/open-telemetry/community/blob/main/projects/llm-semconv.md). These roles are more about the project of expansion and do not affect the maintainer/approver roles in the LLM SemConv SIG which iiuc is governing this. Do I understand this right?

Reply:
right, project leads do not get maintainer/approvers status, but usually have one or both because they are usually deeply familiar with the project and interested in moving it forward.